src/app/base/components/comment/comment.component.html

```diff
@@ -167,14 +170,19 @@
                 style="line-height:1rem;border-radius: 0px" (change)="preventEventPropagation($event)"
                 (keyup)="preventEventPropagation($event)" (keydown)="preventEventPropagation($event)"></textarea>
             <div class="flex flex-row mt-2 justify-between">
-                <button (click)="saveCommentToDb(commentBox.value,privateComment.checked);commentBox.value=''"
-                    class="px-2.5 py-1.5 border border-gray-300 shadow-sm text-xs font-semibold rounded-md text-gray-700 bg-white hover:bg-gray-50 focus:outline-none"
-                    [ngClass]="commentBox.value =='' || newComment.commentId == '' ? 'opacity-50 cursor-not-allowed pointer-events-none' : 'opacity-100 cursor-pointer'">
-
+                <button *ngIf="commentBox.value =='' || newComment.commentId == '' else enabledSave"
+                    class="px-2.5 py-1.5 border border-gray-300 opacity-50 cursor-not-allowed pointer-events-none shadow-sm text-xs font-semibold rounded-md text-gray-700 bg-white hover:bg-gray-50 focus:outline-none">
```

Comment:
'pointer-events-none' class prevents the 'cursor-not-allowed', so this could be solved by removing the 'pointer-events-none'  and adding disabled